FlowCal/mef.py

```diff
@@ -182,37 +237,38 @@ def selection_std(populations,
     # deviation of almost zero. This is an effect of the finite bin resolution
     # and probably gives a bad estimate of the standard deviation. We choose
     # to be conservative and overestimate the standard deviation in these
-    # cases. Therefore, we set the minimum standard deviation to one.
-    min_std = 1.0
+    # cases. Therefore, we set the minimum standard deviation to 0.005.
+    min_std = 0.005
     pop_std[pop_std < min_std] = min_std
 
     # Return populations that don't cross either threshold
-    selected_mask = np.logical_and((pop_mean - n_std_low*pop_std) > low,
-                                   (pop_mean - n_std_high*pop_std) < high)
+    selected_mask = np.logical_and(
+        (pop_mean - n_std_low*pop_std) > low,
+        (pop_mean + n_std_high*pop_std) < high)
     return selected_mask
 
-def fit_beads_autofluorescence(fl_channel, fl_mef):
+def fit_beads_autofluorescence(fl_au, fl_mef):
     """
     Fit a standard curve using a beads model with autofluorescence.
 
     Parameters
     ----------
-    fl_channel : array
-        Fluorescence values of bead populations in channel units.
+    fl_au : array
+        Fluorescence values of bead populations in arbitrary units.
     fl_mef : array
         Fluorescence values of bead populations in MEF units.
 
     Returns
     -------
     std_crv : function
-        Standard curve that transforms arbitrary fluorescence values from
-        channel units to MEF units. This function has the signature ``y =
-        std_crv(x)``, where `x` is some fluorescence value in channel units
-        and `y` is the same fluorescence expressed in MEF units.
+        Standard curve that transforms arbitrary fluorescence values to MEF
+        units. This function has the signature ``y = std_crv(x)``, where
+        `x` is some fluorescence value in arbitrary units and `y` is the
+        same fluorescence expressed in MEF units.
```

Comment:
saw one instance where it looked like you accidentally replaced `fl_au` with `fl_mef` (instead of `fl_rfi`). noted it in a separate comment.